CIP-0102/README.md

```diff
@@ -150,10 +151,9 @@ In addition to providing a way to create guaranteed royalties, this has several
 - Minimal Storage Requirement - An optional boolean has about the smallest memory impact possible. This is especially important because it's attached to the - Reference NFT and will be set for each individual NFT.
 - Intra-Collection Utility - This already allows for minting a collection with some NFTs with royalties and some without. A future version of this standard will likely make use of this field to allow for multiple versions of royalties for even more granular control.
 
-
 ### Backward Compatibility
-To keep metadata compatibility with changes coming in the future, we introduce a `version` field in the datum.
 
+To keep metadata compatibility with changes coming in the future, we introduce a `version` field in the datum.
 
```

Comment:
that's OK; merging it now would be less awkward than having to belay merging this PR because of the link not being valid yet.  I think @Ryun1 meant to approve that soon but we can't do anything with it until @Crypto2099 resolves the current merge conflict.  So in the meantime I think this commit makes sense because it reflects how things _should be_. 😎